app/gui/src/project-view/components/GraphEditor/widgets/WidgetTableEditor.vue

```diff
@@ -194,10 +197,13 @@ function processDataFromClipboard({ data, api }: ProcessDataFromClipboardParams<
   const focusedCell = api.getFocusedCell()
   if (focusedCell === null) console.warn('Pasting while no cell is focused!')
   else {
-    pasteFromClipboard(data, {
+    const pasted = pasteFromClipboard(data, {
       rowIndex: focusedCell.rowIndex,
       colId: focusedCell.column.getColId(),
     })
+    if (pasted.rows < data.length || pasted.columns < (data[0]?.length ?? 0)) {
+      pasteWarning.show(`Truncated pasted data to keep table within ${CELLS_LIMIT} limit`)
```

Comment:
If I understand correctly, the message displayed to the user will be
> Truncated pasted data to keep table within 256 limit

256 of _what_? From context of course we know its the number of cells, but for the user it is not at all clear. It could be columns, rows, _perhaps bytes?_ or other stuff. I think we need to phrase the message to be clearer.